codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/plugins/DefaultAWSAuthSchemePlugin.kt

```diff
@@ -1,7 +1,8 @@
 package software.amazon.smithy.aws.swift.codegen.plugins
 
```

Comment:
Changes in this file are just adding `bearerTokenIdentityResolver` field as one of the new fields configurable by the auth scheme plugin.